### PR TITLE
add clear search query button

### PR DESCRIPTION
**Pull request by AMBacelar**
## Who is this for?
Any user of the search page

## What is it doing for them?
allowing for an easy way to clear the search query

### CODE REVIEW

**Comment by AMBacelar** [comment]
A conversation can be had with regards to what exactly we want to occur when the query is cleared.
The current behaviour matches the behaviour of the current works/image search, which is simply emptying the query.

The issues I have with this are that, no new query is made, so after clicking a button that "Clears the search query" the search has not updated, instead all that has happened is that the input field is cleared, when a user may anticipate that the whole search section is reset, filters and all.
This of course is part of the conversation on #8774 where a discussion on the default state of those pages will not include any results.

**Comment by rcantin-w** [comment]
> The issues I have with this are that, no new query is made, so after clicking a button that "Clears the search query" the search has not updated, instead all that has happened is that the input field is cleared, when a user may anticipate that the whole search section is reset, filters and all.

Maybe! But I think it's also that it's useful on mobile (or possibly even for a11y reasons!) to be able to clear your search quickly when you want to try something different. Maybe @alicerichmond would have insight on this?

File: common/views/components/SearchBar/SearchBar.tsx [hunk 1]
```diff
@@ -64,6 +65,17 @@ const SearchBar: FunctionComponent<{ type: string }> = ({ type }) => {
             placeholder="search..."
             ariaLabel={`Search ${type}`}
           />
+          {inputQuery && (
+            <ClearSearch
+              inputRef={searchInput}
+              setValue={setInputQuery}
+              gaEvent={{
```
Comment (rcantin-w) [inline common/views/components/SearchBar/SearchBar.tsx:72]: Again, is this something we'd want to track 🤔 Do we have a list somewhere of these?

**Review by rcantin-w — APPROVED** [review]
Questions and comments but not blockers ⭐

**Pull request merged**